src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -176,11 +177,31 @@ function WebGLShadowMap( _renderer, _lights, _objects, capabilities ) {
 				_shadowMapSize.x *= 4.0;
 				_shadowMapSize.y *= 2.0;
 
+
+				// for point lights we set the shadow matrix to be a translation-only matrix
+				// equal to inverse of the light's position
+
+				var elements = light.matrixWorld.elements;
+				shadowMatrix.makeTranslation( - elements[ 12 ], - elements[ 13 ], - elements[ 14 ] );
```

Comment:
Instead of creating `shadowMatrix` declaratively as before, I create it directly. Avoids an intermediate Vector3 but loses clarity. Is an intermediate Vector3 preferable after all?